src/VirtualClient/VirtualClient.Dependencies/RedisPackageInstallation.cs

```diff
@@ -154,5 +160,35 @@ await this.systemManager.PackageManager.RegisterPackageAsync(redisPackage, cance
                             .ConfigureAwait(false);
 
         }
+
+        private async Task InstallOnAzLinuxAsync(EventContext telemetryContext, CancellationToken cancellationToken)
```

Comment:
I believe this would also work on RHEL/Fedora distros?

Reply:
Yes, but I haven't tested these cases. Should I also include RHEL/Fedora distros?

Reply:
I think so, Redis is important enough that we need more than ubuntu and AzLinux